spec/02-integration/18-hybrid_rpc/01-rpc_spec.lua

```diff
@@ -56,9 +57,12 @@ for _, strategy in helpers.each_strategy() do
           -- TODO: perhaps need a new test method
           for _, v in pairs(json.data) do
             if v.ip == "127.0.0.1" and v.rpc_capabilities and #v.rpc_capabilities ~= 0 then
-              table.sort(v.rpc_capabilities)
               assert.near(14 * 86400, v.ttl, 3)
+              assert.matches("^(%d+%.%d+)%.%d+", v.version)
```

Comment:
We need to ensure it's the right version. Let's come back to this after we have the mock DP

Reply:
done.